src/improvements/tasks/eth/004-opcm-upgrade-v300-base/config.toml

```diff
@@ -0,0 +1,28 @@
+templateName = "OPCMUpgradeV300"
+
+[[l2chains]]
+chainId = 8453
+name = "Base"
+
+[[opcmUpgrades]]
+chainId = 8453
+# https://www.notion.so/oplabs/Upgrade-14-MTCannon-1d6f153ee1628024af26cd0098d3bdfe
+absolutePrestate = "0x03ee2917da962ec266b091f4b62121dc9682bb0db534633707325339f99ee405"
+expectedValidationErrors = "PROXYA-10,SYSCON-20,DF-30,PDDG-DWETH-30,PDDG-DWETH-40,PDDG-ANCHORP-40,PDDG-120,PLDG-DWETH-30,PLDG-DWETH-40,PLDG-ANCHORP-40"
+
+[addresses]
+OPCM = "0x3a1f523a4bc09cd344a2745a108bb0398288094f" # https://github.com/ethereum-optimism/superchain-registry/blob/b3d020de42abeebeb5786ea5508aa08d12bdf4cd/validation/standard/standard-versions-mainnet.toml#L22C56-L22C98
+StandardValidatorV300 = "0x1b8f0e76bb49b60dffbe3b847e25d6429ebff3e6" # https://github.com/ethereum-optimism/optimism/blob/f79ed8b9c9cbdbf8bb492074f3f98da7f072e21a/op-validator/pkg/validations/addresses.go#L22C37-L22C79
```

Comment:
Is this the latest `StandardValidator300`?  I don't think we should be hitting the `SYSCON-20` error.

Looks like Base's gas limit is at 120,000,000:
```
% cast call 0x73a79Fab69143498Ed3712e519A88a918e1f4072 "gasLimit()(uint64)" -r mainnet
120000000 [1.2e8]
```

Which should be [in bounds](https://github.com/ethereum-optimism/optimism/blob/435056d91850aa49599b1b7e0ac2683c63926be9/packages/contracts-bedrock/src/L1/StandardValidator.sol#L181).

@maurelian - do we need a new deployment that includes [these changes](https://github.com/ethereum-optimism/optimism/pull/15015)?

Reply:
Same issue with the `-DWETH-40` errors.  Seems like those should match the [new standard value (302400)](https://github.com/ethereum-optimism/optimism/blob/273a0da4424b2db3e8f600f42a98dbbf5f39f9a4/op-deployer/pkg/deployer/standard/standard.go#L23-L23).  It looks like the existing WETH contracts do match, but we're still getting errors.